### PR TITLE
Just a way to isolate methods for better readability

### PR DESCRIPTION
**Pull request by duranmla**
# Context
As we got some JavaScript within our website I want to have a minimal organization to be able to read easily what is going on there.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,57 +1,84 @@
...
+    var that = this;
+
+    this.mobileMenuToggle.on('click', function(event) {
+      that.mobileMenuElements.toggleClass('active');
+    });
+
+    this.mobileNavLinks.on('click', function(event) {
+      that.mobileMenuElements.removeClass('active');
+      that.linkWasClicked = true;
+      event.preventDefault();
```
Comment (TheMightyPenguin) [inline js/main.js]: This was my bad really, but could you remove this line (77) @duranmla ? This was when i was trying to implement to scroll, but I moved to do something else and forgot to remove this line!

Comment (duranmla) [reply js/main.js]: Sure

**Comment by duranmla** [comment]
@TheMightyPenguin I am not able to connect to slack please check this out

**Comment by TheMightyPenguin** [comment]
LGTM

File: js/main.js (hunk 1 above)
Comment (TheMightyPenguin) [inline js/main.js]: LGTM

**Review by TheMightyPenguin — APPROVED** [review]
LGTM

**Pull request merged**